components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/ServerApplicationManagementService.java

```diff
@@ -115,6 +123,51 @@ public ApplicationModel getApplication(String applicationId) {
         }
     }
 
+    /**
+     * Export an application identified by the applicationId, as an XML string.
+     *
+     * @param applicationId ID of the application to be exported.
+     * @param exportSecrets If True, all hashed or encrypted secrets will also be exported.
+     * @return XML string of the application.
+     */
+    public String exportApplication(String applicationId, Boolean exportSecrets) {
+
+        try {
+            String tenantDomain = ContextLoader.getTenantDomainFromContext();
+            return getApplicationManagementService().exportSPApplicationFromAppID(
```

Comment:
Have we set a default value for exportSecrets at the swagger level?
We might need to do a null check for 'exportSecrets' before passing it to OSGi service. Otherwise will result in a NPE.

Reply:
Just noticed that you have done that at swagger level